### PR TITLE
Install gsutil.

### PR DESCRIPTION
**Pull request by aaron276h**

### CODE REVIEW

File: Dockerfile.cpu [hunk 1]
```diff
@@ -69,3 +69,8 @@ RUN ${CONDA_DIR}/bin/python3.6 -m pip install -r files/cli-requirements.txt
 COPY files/additional-requirements.txt files/additional-requirements.txt
 RUN ${CONDA_DIR}/bin/python3.6 -m pip install -r files/additional-requirements.txt
 ENV JUPYTER_CONFIG_DIR=/run/pedl/jupyter/config JUPYTER_CONFIG_DIR=/run/pedl/jupyter/config JUPYTER_DATA_DIR=/run/pedl/jupyter/data JUPYTER_RUNTIME_DIR=/run/pedl/jupyter/runtime
+
+RUN apt-get update -y && echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+RUN apt-get install apt-transport-https ca-certificates gnupg -y
+RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+RUN apt-get update -y && apt-get install google-cloud-sdk -y
```
Comment (brain-good) [inline Dockerfile.cpu]: can we move this install to line 62 -- same respective location for the GPU dockerfile

Comment (aaron276h) [reply Dockerfile.cpu]: Done

File: Dockerfile.gpu [hunk 2]
```diff
@@ -62,6 +62,12 @@ RUN mkdir -p /build \
      ./configure \
   && make install
 
+RUN apt-get update -y && echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+RUN apt-get install apt-transport-https ca-certificates gnupg -y
+RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+RUN apt-get update -y && apt-get install google-cloud-sdk -y
+
+
```
Comment (brain-good) [inline Dockerfile.gpu]: delete one of these lines

Comment (aaron276h) [reply Dockerfile.gpu]: done

**Pull request merged**